README.MD

```diff
@@ -28,10 +27,19 @@ import { render } from 'react-dom'
 import { createStore, combineReducers } from 'redux';
 
 // 1. Connect your data, similar to react-redux @connect
+// Decorator version
 @asyncConnect({
-  lunch: (params, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
+  lunch: (params, store, helpers) => Promise.resolve({id: 1, name: 'Borsch'})
 })
 class App extends React.Component {
+  // 1. Same as the decorator but as a function
+  static reduxAsyncConnect(params, store, helpers) {
+    const {dispatch, getState} = store;
+    if (!helpers.isLoaded(getState())) {
+      return Promise.all([dispatch(loadWidgets())]);
```

Comment:
i'll clarify the examples (except the one on the server as i don't use node there myself so have no idea what it's supposed to be)